app/models/conference.rb

```diff
@@ -27,14 +24,10 @@ def date_range
     @date_range ||= DateRange.new(start_date: starts_on, end_date: ends_on)
   end
 
-  def chronological_dates
-    unless starts_on <= ends_on
-      errors.add(:ends_on, 'must be a later date than start date')
-    end
-  end
+  private
 
-  def tickets_left
```

Comment:
Is it no longer used?

I just removed this method because it wasn't used any more...

Reply:
I think there is probably more that should be removed, related to this. Maybe turn it into an issue?

Reply:
I'm afraid I don't get it 🙈 

What should be removed? The number of tickets is still displayed somewhere and the column apparently a thing. I just removed this single method because I could not see it being used.

Reply:
As I recall it, the tickets and some other things (accommodation, flights, I think) are not being used since the raffles ceased to be. But the db and relevant code has not been cleaned up.
Just wanted to mention it, because it may be easier to clean everything up at once. 
(seeds, lib/conferences <- for the raffle, specs, to name a few).